### DISCUSSION TITLE
第6节课笔记(12班)

### DISCUSSION THREAD

**Discussion by fanqiNO1** [InternLM/Tutorial#466] [笔记]
xdm，加油！

**Comment by HuangXinzhe** [comment]
https://blog.csdn.net/qq_29787929/article/details/135717533

**Comment by jujimeizuo** [comment]
https://note.jujimeizuo.cn/llm/internlm/lec6/

**Comment by cheng221** [comment]
https://spice-spice-829.notion.site/0c1cf1a3ea18431cb4d187de7062717b?pvs=4

**Comment by ChengaoJ** [comment]
https://blog.csdn.net/weixin_43193528/article/details/135831368

**Comment by canxingliuying** [comment]
https://blog.csdn.net/weixin_42449385/article/details/135797073?spm=1001.2014.3001.5502

**Comment by deseijily** [comment]
https://j1b8l2ps5tm.feishu.cn/docx/Mld6dzmxsogEUIxQcK3chLynnq0

**Comment by ltstar** [comment]
[课程6-OpenCompass.md](https://github.com/InternLM/tutorial/files/14060884/6-OpenCompass.md)
